deviations/tests.py

```diff
@@ -368,6 +368,11 @@ def test_get_deviation_groups(self):
         extra_exercise.delete()
 
     def test_add_deadline_deviations(self):
+        # Fix to issue #993 made this test to fail, because DISTINCT is not supported
+        # See: https://github.com/apluslms/a-plus/issues/1003
+        # Temporarily disabling this test for now, until the issue has been fixed
+        return
```

Comment:
There is actually a `skip` decorator that should be used instead of `return`. The test runs will list the skipped tests in the end so that they are not forgotten.

https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

```
@unittest.skip("Skipping until tests are run on PostgreSQL instead of SQLite")
def test_add_...(self):
    ...
```